draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -340,37 +354,59 @@
           configured with an algorithm to combine the recognized metrics
           carried by the Sub-TLVs within a Metadata Path Attribute of the
           received BGP UPDATE message.</t>
+	  <!-- XXX JMH - This text is really an error handling consideration.
+	  Perhaps consider renaming the section to note that? -->
         </section>
 
         <section title="Metadata Path Attribute Handling Procedure">
           <t>The Metadata Path Attribute MUST contain at least one metadata
           Sub-TLV. Multiple Metadata Sub-TLVs can be included in a Metadata
           Path Attribute in one BGP UPDATE message. The content of the
-          Sub-TLVs present in the BGP Metadata Path attribute is determined by
-          the configuration. When a BGP Speaker does not recognize some of the
-          Sub-TLVs within one Metadata Path Attribute in a BGP UPDATE message,
-          the BGP Speaker should forward the received BGP UPDATE message
-          without any change if the transitive bit is set to 1 <xref
-          target="RFC4271"/>. The domain ingress nodes should process the
+          Sub-TLVs present in the BGP Metadata Path Attribute is determined by
+          configuration.  The domain ingress nodes should process the
           recognized Sub-TLVs carried by the Metadata Path Attribute and
           ignore the unrecognized Sub-TLVs. By default, a BGP speaker does not
           report any unrecognized Sub-TLVs within a Metadata Path Attribute
           unless configured to send a notification to its management system.
           The ingress node should be configured with an algorithm to combine
           the recognized metrics carried by the Sub-TLVs within a Metadata
           Path Attribute of the received BGP UPDATE message.</t>
+	  <!-- XXX JMH "combine" is odd.  Perhaps utilize? -->
+
+	  <!-- XXX JMH - the text below doesn't make much bgp sense.  Path
+	  attributes apply to all of the NLRI carried in the update message.  I
+	  recommend striking this section and have commented it out to flag
+	  doing so.
 
           <t>The metrics Sub-TLVs included in the Metadata Path Attribute
-          apply to all the address families carried in the NLRI field of the
+          apply to all of the address families carried in the NLRI field of the
```

Comment:
Instead of saying "apply to all the address families", is "can be attached to all the address families" making more BGP sense? 

The metrics Sub-TLVs included in the Metadata Path Attribute can be attached all the address families
carried in the NLRI field of the BGP UPDATE message . For a multi-protocol BGP
UPDATE message , the metrics Sub-TLVs included in the Metadata Path
Attribute apply to all the AFIs/SAFIs address families carried by the MP_REACH_NLRI.

Reply:
Again, this is bgp fundamentals and it doesn't need to be re-stated.

By comparison, if you were to tell me that the metadata applied inconsistently for NLRI and MP-REACH-NLRI, I'd tell you the feature is outright broken and to try again.

Reply:
We can delete this paragraph.

Reply:
Removed in push.